client/src/apps/chord-recognizer/src/index.js

```diff
@@ -25,13 +25,14 @@ export default function ChordRecognizer({chords}) {
         }), 4000)
       }
       // add new detections
-      if ((!prev || prev.id !== chords.id) && chords.detection.length) {
+      if ((!prev || prev.id !== chords.id) && chords.detection && chords.detection.length) {
```

Comment:
remove that, it shouldn't happen anymore

Reply:
still happening, please remove this when you fix it